hello.py

```diff
@@ -0,0 +1,2 @@
+print("hello world!")
+print("also go to f")
```

Comment:
future off with this line
```suggestion
print("also go to future")
```

Reply:
no